### PR TITLE
Updated README.md added Othman Edziza

### PR DESCRIPTION
**Pull request by ersuamine**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <h1 align="center">
 <img src="https://us.zonerama.com/photos/1006679669_2484x471.jpg" />
 </h1>
+
 # Our Team
 
 ## Dilovar Turaev
```
Comment (dilovar1992) [inline README.md:7]: i see my name good

Comment (dilovar1992) [reply README.md:7]: good of you to add Osman

Comment (4745Mthn) [reply README.md:7]: Looks fantastic ,

**Review by 4745Mthn — APPROVED** [review]

**Review by greengreene — COMMENTED** [review]
Hello

**Pull request merged**